### PR TITLE
Added functionality for the planner to block execution of a job until in...

### PR DESCRIPTION
**Pull request by ibuenros**
...put is ready.

### CODE REVIEW

**Comment by ibuenros** [comment]
@liyinan926 can you review?

File: camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeperPlanner.java [hunk 1]
```diff
@@ -86,4 +86,14 @@ protected String pathListToCommaSeperated(List<Path> list) {
...
     return this.outlierProperties;
   }
+
+  /**
+   * Blocks processing of a job until the input is ready.
+   * By default, will return immediately and proceed with the job.
+   * @param jobProps Job properties.
+   * @return true to proceed with the job, false to cancel the job.
+   */
+  public boolean waitUntilReadyToProcess(Properties jobProps) {
```
Comment (liyinan926) [inline camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeperPlanner.java]: Is this method going to be overridden in a subclass to provide a different behavior? This method can be protected if is not supposed to be overridden outside the package.

Comment (ibuenros) [inline camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeperPlanner.java]: fixed

**Comment by zliu41** [comment]
LGTM

**Pull request merged**